template.tpl

```diff
@@ -523,6 +575,68 @@ ___WEB_PERMISSIONS___
                     "boolean": true
                   }
                 ]
+              },
+              {
+                "type": 3,
+                "mapKey": [
+                  {
+                    "type": 1,
+                    "string": "consentType"
+                  },
+                  {
+                    "type": 1,
+                    "string": "read"
+                  },
+                  {
+                    "type": 1,
+                    "string": "write"
+                  }
+                ],
+                "mapValue": [
+                  {
+                    "type": 1,
+                    "string": "ad_user_data"
+                  },
+                  {
+                    "type": 8,
+                    "boolean": true
```

Comment:
Should this be true and true or true and false?

Reply:
This is the same change as the other comment, just for the `ad_user_data` permission (the GTM template needs permission to both read (for the unit test) & write (to set defaults) the consent values)